### PR TITLE
Change tabs to spaces

### PR DESCRIPTION
**Pull request by MichaelDeciso**
As requested by @fichtner .

### CODE REVIEW

File: src/etc/inc/gwlb.inc [hunk 1]
```diff
@@ -33,15 +33,15 @@
 function return_dpinger_defaults()
```
Comment (fichtner) [inline src/etc/inc/gwlb.inc]: keep these... only leading tabs please (^\t)

File: src/etc/inc/system.inc [hunk 2]
```diff
@@ -721,16 +721,16 @@ function system_syslogd_start($verbose = false, $restart = false)
     if (!isset($syslogcfg['disablelocallogging'])) {
```
Comment (fichtner) [inline src/etc/inc/system.inc]: keep these, too

File: src/www/diag_backup.php [hunk 3]
```diff
@@ -70,7 +70,7 @@ function restore_config_section($section_name, $new_contents)
 }
```
Comment (fichtner) [inline src/www/diag_backup.php]: keep :)

File: src/etc/inc/gwlb.inc (hunk 1 above)
Comment (MichaelDeciso) [reply src/etc/inc/gwlb.inc]: That violates PSR-2, which says spaces should be used throughout. (And even most tab-intended code bases use spaces for aligning comments.)

Comment (MichaelDeciso) [reply src/etc/inc/gwlb.inc]: Ok, I will revert them for now.

I really hope the PHP files will be PSR-2 compliant sooner than later, though. I like the code base to be clean and consistent.

**Comment by fichtner** [comment]
Thanks!

**Pull request merged**